installer/templates/new/jobs/phrases.exs

```diff
@@ -3,5 +3,5 @@ use Kitto.Job.DSL
 job :text, every: {4, :seconds} do
```

Comment:
Rename the job to `:phrases` and remove the 1st argument which isn't required anymore from broadcast.